openshift/e2e-tests-openshift.sh

```diff
@@ -83,83 +83,105 @@ function timeout() {
   return 0
 }
 
-function install_knative(){
-  header "Installing Knative serving"
+function install_servicemesh(){
+  header "Installing ServiceMesh"
 
-  create_knative_namespace serving
+  # Install the ServiceMesh Operator
+  oc apply -f openshift/servicemesh/operator-install.yaml
 
-  echo ">> Patching Knative Serving CatalogSource to reference serving release ${SERVING_RELEASE_BRANCH}"
-  RELEASE_YAML="https://raw.githubusercontent.com/openshift/knative-serving/${SERVING_RELEASE_BRANCH}/openshift/release/knative-serving-${SERVING_RELEASE_TAG}.yaml"
-  sed "s|--filename=.*|--filename=${RELEASE_YAML}|"  openshift/olm/knative-serving.catalogsource.yaml > knative-serving.catalogsource-ci.yaml
+  # Wait for the istio-operator pod to appear
+  timeout 900 '[[ $(oc get pods -n openshift-operators | grep -c istio-operator) -eq 0 ]]' || return 1
 
-  # Install CatalogSources in OLM namespace
-  oc apply -n $OLM_NAMESPACE -f knative-serving.catalogsource-ci.yaml
-  timeout 900 '[[ $(oc get pods -n $OLM_NAMESPACE | grep -c knative) -eq 0 ]]' || return 1
-  wait_until_pods_running $OLM_NAMESPACE
+  # Wait until the Operator pod is up and running
+  wait_until_pods_running openshift-operators || return 1
 
-  # Deploy Knative Operators Serving
-  deploy_knative_operator serving KnativeServing
+  # Deploy ServiceMesh
+  oc new-project $SERVICEMESH_NAMESPACE
+  oc apply -n $SERVICEMESH_NAMESPACE -f openshift/servicemesh/controlplane-install.yaml
```

Comment:
I'd not vendor those files, if not really really needed:
https://github.com/openshift/knative-eventing/blob/release-v0.8.0/openshift/e2e-tests-openshift.sh#L54-L64